File: client/src/app/Models/Employee.ts
import { Role } from './Role';

export class Employee{
    employee_ID:number;
    employee_Name:string;
    gender:boolean;
    address:string;
    telePhone:string;
    dateOfBirth:Date;
    role!: Role;
    token?: string;
    constructor(employee_ID:number,employee_Name:string,gender:boolean,address:string,telePhone:string,dateOfBirth:Date)
    {
        this.employee_ID=employee_ID;
        this.employee_Name=employee_Name;
        this.gender=gender;
        this.address=address;
        this.telePhone=telePhone;
        this.dateOfBirth=dateOfBirth;
    }
}

File: webapi/StoreManagement/Models/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace StoreManagement.Models
{
    public class Employee
    {
        public int Employee_ID { get; set; }
        public string Employee_Name { get; set; }
        public Boolean Gender { get; set; }
        public string Address { get; set; }
        public string TelePhone { get; set; }
        public DateTime DateOfBirth { get; set; }
        public ICollection<Order> Orders { get; set; }
    }
}
